src/components/VisibilityCard/VisibilityCard.tsx

```diff
@@ -106,38 +125,57 @@ export const VisibilityCard: React.FC<VisibilityCardProps> = props => {
             <>
               <p className={classes.label}>
                 {intl.formatMessage({
-                  defaultMessage: "Visible"
+                  defaultMessage: "Published"
```

Comment:
I think this should be passed as prop because it's missing a context.
"Opublikowany" lub "Opublikowana" in polish makes difference